test/test_helper.rb

```diff
@@ -192,6 +192,20 @@ def with_settings(options, &_block)
     saved_settings.each { |k, v| Setting[k] = v }
   end
 
+  REPOSITORY_PATH = Rails.root.to_s.gsub(%r{config\/\.\.}, '') + '/tmp/test/filesystem_repository'
+
+  def with_existing_filesystem_scm(&block)
+    if Dir.exists?(REPOSITORY_PATH)
+      Setting.enabled_scm = Setting.enabled_scm << "Filesystem" unless Setting.enabled_scm.include? "Filesystem"
```

Comment:
Line is too long. [112/99]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.